### PR TITLE
Shubham dalal work

### PR DESCRIPTION
**Pull request by shubhamdalal49**

### CODE REVIEW

File: Shubham_Dalal/HTML/ASSIGNMENT7/Shubham_homepage.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        <h1>Welcome to Job Portal</h1>
+        <header>
+            <ul>
+                <li><a href="#">Home</a> </li>
+                <li><a href="shubham_login.html">Log-in</a></li>
+                <li><a href="Shubham_personal_academic.html">Profile</a></li>
+            </ul>
+        </header>
+        <div>
+            <img src="job.jpg" alt="">
```
Comment (preetipie22) [inline Shubham_Dalal/HTML/ASSIGNMENT7/Shubham_homepage.html]: why alt tag is empty?

File: Shubham_Dalal/HTML/ASSIGNMENT7/Shubham_personal_academic.html [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+                            <th>Name</th>
+                            <td><input type="text" placeholder="enter your name" class="input"/></td>
+                        </tr>
+                        <tr>
+                            <th>E-mail</th>
+                            <td><input type="email" placeholder="enter your email" class="input"/></td>
+                        </tr>
+                        <tr>
+                            <th>Mobile</th>
+                            <td><input type="number" placeholder="enter your mobile" class="input"/></td>
```
Comment (preetipie22) [inline Shubham_Dalal/HTML/ASSIGNMENT7/Shubham_personal_academic.html]: for mobile number input type should be 
`<input type="tel">`

File: Shubham_Dalal/HTML/ASSIGNMENT7/shubham_professionl.html [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+        <div class="container">
+            <img src="job.jpg" alt="">
+            <div class="box">
+                <table>
+                    <tr>
+                        <td><h2>Profession Details</h2></td>
+                    </tr>
+                    <tr>
+                        <th class="same">Work Experience</th>
+                        <td><input type="number"></td>
```
Comment (preetipie22) [inline Shubham_Dalal/HTML/ASSIGNMENT7/shubham_professionl.html]: Add min="0" for experience like
`<td><input type="number" min="0"></td>`

**Review by preetipie22 — CHANGES_REQUESTED** [review]
Please make the requested changes

**Review by preetipie22 — CHANGES_REQUESTED** [review]
changes requested

**Review by Ashwin2812 — APPROVED** [review]

**Review by nileshp1122 — APPROVED** [review]

**Review by preetipie22 — APPROVED** [review]

File: Shubham_Dalal/HTML/ASSIGNMENT7/assignment7_login.css [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+    padding: 20px;
+}
+li a:hover {
+    background-color: teal;
+}
+.center {
+    display: flex;
+    justify-content: center;
+}
+div img {
```
Comment (SnehaMakade) [inline Shubham_Dalal/HTML/ASSIGNMENT7/assignment7_login.css:34]: if there are same classes then u can use the above style sheet why repeated classes in another file

File: Shubham_Dalal/HTML/ASSIGNMENT7/assignment7_personal.css [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+header ul li a {
+    text-decoration: none;
+    color: white;
+    text-align: center;
+    padding: 20px;
+}
+li a:hover {
+    background-color: teal;
+}
+div img {
```
Comment (SnehaMakade) [inline Shubham_Dalal/HTML/ASSIGNMENT7/assignment7_personal.css:30]: again till this class same classes?

File: Shubham_Dalal/HTML/ASSIGNMENT7/shubham_login.html [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+                <li><a href="Shubham_homepage.html">Home</a></li>
+                <li><a href="#">Log-in</a></li>
+                <li><a href="shubham_professionl.html">Profile</a></li>
+            </ul>
+        </header>
+        <div class="center">
+            <img src="job.jpg" alt="job portal" />
+            <div class="log">
+                <label class="same">E-mail</label>
+                <input type="text" placeholder="enter email" class="same">
```
Comment (SnehaMakade) [inline Shubham_Dalal/HTML/ASSIGNMENT7/shubham_login.html]: placeholders 1st letter should be capital

File: Shubham_Dalal/HTML/ASSIGNMENT7/shubham_login.html [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+                <label class="same">password</label>
```
Comment (SnehaMakade) [inline Shubham_Dalal/HTML/ASSIGNMENT7/shubham_login.html]: label should be start with capital

**Review by SnehaMakade — CHANGES_REQUESTED** [review]
check with classes repeating with 3 files and do changes accordingly

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**